model/ContainerRuntime.php

```diff
@@ -63,11 +63,13 @@ public function getDeliveryContainer($deliveryId)
     public function getRuntime($deliveryId)
     {
         $delivery = $this->getResource($deliveryId);
-        if (!$delivery->exists()) {
+        $runtimeResource = $delivery->getOnePropertyValue($this->getProperty(self::PROPERTY_RUNTIME));
+        if (is_null($runtimeResource)) {
```

Comment:
```suggestion
        if ($runtimeResource === null) {
```

Reply:
Out of curiosity: What difference would that make?

Reply:
info : performance guide tip offered by PhpStorm IDE
A: it is a micro performance enhancement

Reply:
Thanks, Mike!